shopinvader/services/customer.py

```diff
@@ -35,6 +38,24 @@ def create(self, **params):
         self._post_create(self.work.partner)
         return self._prepare_create_response(binding)
 
+    def update(self, _id, **params):
+        # We get the binding from the partner
+        if not self.partner:
+            raise UserError(_("You must provide a partner"))
+
+        partner = self._get(_id)
+        if not partner.shopinvader_bind_ids:
+            raise UserError(_("Partner %s is not a customer") % partner.id)
+        binding = partner.shopinvader_bind_ids[0]
```

Comment:
you should filter on current backend. 
```suggestion
        partner = self._get(_id)
        binding = partner._get_invader_partner(self.shopinvader_backend)
        if not binding:
            raise UserError(_("Partner %s is not a customer") % partner.id)
```